langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -357,6 +509,19 @@ public isolated function forEach(xml<ItemType> x, @isolatedParam function(ItemTy
 #
 # Each item is represented as a singleton value.
 #
+# ```ballerina
+# xml codingBooks = xml `<scripting>JS</scripting><programming>Ballerina</programming>
+#                   <programming>Python</programming><scripting>Perl</scripting>`;
+# 
+# xml programmingBooks = codingBooks.filter(function (xml xmlItem) returns boolean {
+#       if (xmlItem is xml:Element) {
```

Comment:
```suggestion
#       if xmlItem is xml:Element {
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/7b4dfa608b04064393e46211524556d9850dfddf